multiversx_sdk_cli/simulation.py

```diff
@@ -1,13 +1,16 @@
 from collections import OrderedDict
 from typing import Any, Dict, Protocol
 
-from multiversx_sdk_cli.utils import ISerializable
 from multiversx_sdk import Transaction, TransactionOnNetwork
 
+from multiversx_sdk_cli.utils import ISerializable
+
 
+# fmt: off
```

Comment:
Needed?

Reply:
it is needed for `black` to not format the code, because it moves the three dots on the same line and the `flake` complains